VideoLocker/src/main/java/org/edx/mobile/util/SecurityUtil.java

```diff
@@ -13,20 +12,31 @@
 import java.io.File;
 import java.util.Collections;
 
-public abstract class SecurityUtil {
+/**
+ * Utility class that deals with the security of a user's personal information data.
+ */
+public class SecurityUtil {
     private static final Logger logger = new Logger(SecurityUtil.class);
 
+    private SecurityUtil() {
```

Comment:
Might as well throw an exception here as well, to make this impossible to instantiate successfully.